code/game/objects/structures/window.dm

```diff
@@ -641,6 +671,10 @@
 	explosion_block = 1
 	glass_type = /obj/item/stack/sheet/rglass
 
+/obj/structure/window/full/reinforced/polarized
+	name = "electrochromic window"
+	desc = "Adjusts its tint with voltage. Might take a few good hits to shatter it."
+	var/id
```

Comment:
A space after this block would be nice so it's not squished against the tinted window block below.